From 2a2119e10ca364d7c1539a9103b9fd3839dbe6ac Mon Sep 17 00:00:00 2001
From: Atish Patra <atish.patra@wdc.com>
Date: Thu, 5 Mar 2020 16:24:21 -0800
Subject: [PATCH 1/3] lib: kconfig: Add option to set BZIP2 compression method

commit 2a2119e10ca364d7c1539a9103b9fd3839dbe6ac upstream

There is no way to select BZIP2 compression method.
Add it under library/compression config where all other
compression related configs are present.

Upstream-Status: Backport

Signed-off-by: Atish Patra <atish.patra@wdc.com>
Reviewed-by: Tom Rini <trini@konsulko.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 lib/Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/lib/Kconfig b/lib/Kconfig
index 452f390c80..144a54da28 100644
--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -409,6 +409,11 @@ config GZIP
 	help
 	  This enables support for GZIP compression algorithm.
 
+config BZIP2
+	bool "Enable bzip2 decompression support"
+	help
+	  This enables support for BZIP2 compression algorithm.
+
 config ZLIB
 	bool
 	default y
-- 
2.34.1

